src/cljs/zetawar/views.cljs

```diff
@@ -384,7 +384,12 @@
                                  :on-hide hide-settings}
      [:> js/ReactBootstrap.Modal.Header {:close-button true}
       [:> js/ReactBootstrap.Modal.Title
-       (translate :configure-faction-title)]]
+       (str "Configure Faction: "
```

Comment:
We should still use translate for the title. We should also add translations for the colors (:blue-name, :red-name, etc.) and use them to lookup the color names.